docs/modules/ROOT/pages/how-to/use-change-data-capture.adoc

```diff
@@ -0,0 +1,68 @@
+[[change-data-capture]]
+:description: This page describes how use Change Data Capture API.
+= Use Change Data Capture
+
+link:https://neo4j.com/docs/cdc/current/[Change Data Capture] (CDC) queries can be created with the Cypher Builder by using the Procedures in the `cdc` namespace:
+
+* `Cypher.cdc.current()`
+* `Cypher.cdc.earliest()`
+* `Cypher.cdc.query(from, selectors)`
+
+
+== Examples
+
+=== Acquiring the current change identifier
+
```

Comment:
It would be nice to have more description of what that means, something like 

```suggestion

To acquire the current change identifier, follow these instructions:

```